### ISSUE TITLE
It says "CORS Enabled" but can't access files from external domains. 

### ISSUE THREAD

**Issue by maboumra** [ritwickdey/vscode-live-server#249]
I really don't know if this is a bug report or a feature request because I don't know what you mean by "CORS Enabled" in the description and I didn't find any reference in the docs. Anyway, it would be nice to have Allow Origin headers automatically added to pages served with Live Server. I know that you can add the flags or install an extension to Chrome but that makes navigation while developing insecure. "Allow Origin" should be added only for the pages you are developing.

Chrome 
Live Server 5.3.1

**Comment by JonathanWilbur** [comment]
I agree. The use case with Live Server does not warrant Same Origin Policy. CORS should be enabled.

**Comment by larryhu** [comment]
I found a way to fix this problem so far (even though it looks like so bad),  adding below config to your settings.json .

<pre>
{
    "liveServer.settings.proxy": {
        "enable": true,
        "baseUri": "/api", 
        "proxyUri": "http://127.0.0.1:8080/"
    }
}
</pre>

**Comment by sekti92** [comment]
Hey I have the same issue when using d3.js. How to fix this issue?

```
d3.html('http://enable-cors.org')
  .get((err, data) => {
    console.log(data)
  })
```
![image](https://user-images.githubusercontent.com/14885417/119052092-42436480-b9ee-11eb-9da3-8bc2264ee742.png)
